manifests/d/dimkohoncharov/Elixir/1.15.7/dimkohoncharov.Elixir.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: dimkohoncharov.Elixir
+PackageVersion: 1.15.7

```

Comment:
```suggestion
PackageVersion: 1.15.7
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Erlang.ErlangOTP
      MinimumVersion: 26.0.0
```
If the dependency is missing, the application will not install, but will still give a successful return code
![image](https://github.com/microsoft/winget-pkgs/assets/12611259/a55fe119-1612-4ff5-9c4e-0b0022f4f7c0)